src/infrastructure.sh

```diff
@@ -1292,6 +1292,44 @@ rlServiceRestore() {
     return $failed
 }
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# rlServiceRestore
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+: <<'=cut'
+=pod
+
+=head3 rlServiceStatus
+
+Print status of given C<service>.
+
+    rlServiceStatus service [service...]
+
+=over
+
+=item service
+
+Name of the service(s) to get the state.
+
+=back
+
+Returns status the last service.
```

Comment:
It's not clear now this maps to the integer value of exit status.

Also, grammar.. :)

Reply:
This needs definitely to be fixed. The final fix will vary depending on the discussion above.